go.mod

```diff
@@ -1,14 +1,13 @@
-module github.com/andreani-publico/go-platform
+module github.com/architecture-it/go-platform
 
 go 1.12
 
 require (
+	github.com/architecture-it/go-platform v0.3.6 // indirect
```

Comment:
Está bien esto? Parece como una referencia a su mismo paquete... Me hace ruido

Reply:
si no lo hice yo, ya estaba, según el comentario parece que hace una referencia recursiva

Reply:
eso no estaba *rolling eyes*:
https://github.com/architecture-it/go-platform/blob/master/go.mod

Reply:
jajaj tenes razón.